outernet/federation/v1alpha/federation.proto

```diff
@@ -213,7 +213,7 @@ message InterconnectionPoint {
   // promiscuous mode, or if it can receive L2 packets only if the L2
   // destination address matches it's own physical address.
   enum Mode {
-    MODE_PROMISCUOUS = 0;
+    MODE_PROMISCUOUS_UNSPECIFIED = 0;
```

Comment:
I would suggest the following (I recognize some of these changes go beyond scope of the linting warning).

```
enum PromiscuousMode {
  PROMISCUOUS_MODE_UNSPECIFIED = 0;
  PROMISCUOUS_MODE_ENABLED = 1;
  PROMISCUOUS_MODE_DISABLED = 2;
}
```

- Preserves ability to specify MODE_PROMISCUOUS
- Renames Mode to make UNSPECIFIED more clear (versus MODE_UNSPECIFIED)

Reply:
I changed it. I think we probably should specify how the "unspecified" mode is handled?
Other than this, @electronjoe I think I fixed everything and we can merge.